recipes/prepper/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "prepper" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/varchasgopalaswamy/prepper/archive/v{{ version }}.tar.gz
+  sha256: 5a7f7aa957ba94a36f321c56b4710d2875881b6e5ae9693e93c6b0d8bc890d2a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - h5py >=3.8.0,<3.9
+    - loguru >=0.6.0
+    - aenum >=3.1.11,<3.2
+    - numpy >=1.23.5,<1.24
```

Comment:
This is a somewhat uncommon constraint. Are you sure you need an old version of numpy?